app/mapping/registration/BeneficiariesMapper.scala

```diff
@@ -24,18 +24,23 @@ import play.api.Logger
 class BeneficiariesMapper @Inject()(
                                      individualMapper: IndividualBeneficiaryMapper,
                                      unidentifiedMapper: ClassOfBeneficiariesMapper,
-                                     charityortrustMapper: CharityBeneficiaryMapper
+                                     charityortrustMapper: CharityBeneficiaryMapper,
```

Comment:
I know this isn't pertinent to the work you're doing in this branch but could this be changed to charityMapper? I'm assuming it only works for charity and not both given the class name?

Reply:
Well spotted.